### PR TITLE
feat: gemma2 models samples

### PR DESCRIPTION
**Pull request by gryczj**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [x] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] This pull request is from a branch created directly off of `GoogleCloudPlatform/nodejs-docs-samples`. Not a fork.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: generative-ai/snippets/test/gemma2Prediction.test.js
Comment (rsamborski) [inline generative-ai/snippets/test/gemma2Prediction.test.js:61]: QQ: Can you make one another check to make sure the request format is correct for GPUs and TPUs ([example](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/206a67db56961518750b8bf6e0bd6f10a659f088/gemma2/gemma2_test.py#L65-L80))?

**Review by rsamborski — APPROVED** [review]
LGTM from product perspective with minor ask.
Please wait for a language review as well before merging.

File: generative-ai/snippets/test/gemma2Prediction.test.js
Comment (gryczj) [reply generative-ai/snippets/test/gemma2Prediction.test.js:93]: done, @rsamborski please check https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/3869/commits/3026609ceb4b2467bddbe6db0309f696778ee9f3

Comment (rsamborski) [reply generative-ai/snippets/test/gemma2Prediction.test.js]: LGTM

**Pull request merged**
